### DISCUSSION TITLE
Virtual DOM?

### DISCUSSION THREAD

**Discussion by dannyob** [rabbibotton/clog#102] [Ideas]
I'm not sure if this maps well to CLOG, but one trick modern webapp frameworks use to prevent expensive redraws of the whole web page is Virtual DOM diffing -- see, for example, https://github.com/Matt-Esch/virtual-dom

I wonder if CLOG could benefit from this approach, or an equivalent?

**Comment by rabbibotton** [comment]
Not really. CLOG never redraws the whole of anything and you have better control over the DOM remotely. Thanks though for suggesting.
